src/app/basics/test-suites/recent-runs/page.md

```diff
@@ -92,14 +92,14 @@ Clicking on a recording opens the replay so that you can inspect it with Replay
 {% quick-link
   title="PR comments"
   icon="github"
-  href="/test-suites/pr-comments"
+  href="/basics/test-suites/pr-comments"
   description="The Replay GitHub PR comment makes it easy to view a PR's recent runs."
 /%}
 
 {% quick-link
   title="Top failing and flaky tests"
   icon="sort"
-  href="/test-suites/top-failing-and-flaky-tests"
+  href="/basics/test-suites/top-failing-and-flaky-tests"
   description="Go to the Failing and Flaky Tests view to see your top failing and flaky tests."
 /%}
```

Comment:
Totally don't need to fix this as part of this PR but while I'm noticing it, the screenshots for Recent Runs is outdated

| Shown | Actual |
| :--- | :--- |
| ![image](https://github.com/replayio/docs/assets/29597/ff43cdc0-57fb-4ba3-8b8a-26e766c0e8f8) | ![Screenshot 2024-05-27 at 7 19 11 AM](https://github.com/replayio/docs/assets/29597/23a01167-ec70-423a-bd20-d460da9d547a) |

cc @filiphric

Reply:
noted [DEV-534](https://linear.app/replay/issue/DEV-534/update-recent-runs-image), thanks 👍